docs/extend/index.md

```diff
@@ -118,61 +118,6 @@ remove it, use the `docker plugin remove` command. For other available
 commands and options, see the
 [command line reference](../reference/commandline/index.md).
 
-## Service creation using plugins
```

Comment:
Earlier service creation using plugin filters was part of the generic index file. Services and plugins deserve a separate file, hence the move.